src/main/java/com/example/andersen/Task6/bucket/BucketImpl.java

```diff
@@ -0,0 +1,153 @@
+package com.example.andersen.Task6.bucket;
+
+import com.example.andersen.Task5.exception.PutWrongNumberException;
+import com.example.andersen.Task6.dao.Product;
+import com.example.andersen.Task6.warehouse.Warehouse;
+import org.springframework.stereotype.Service;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static com.example.andersen.Task6.warehouse.Warehouse.checkForExpiration;
+
+@Service
+public class BucketImpl implements Bucket {
+    private final Warehouse warehouse = new Warehouse();
+    private Map<Product, Integer> orders = new HashMap<>();
+
+    public boolean isEnoughAmountOfProduct(int product_id, int wanted_amount) {
+        Map<Product, Integer> map = warehouse.getAllProducts();
```

Comment:
Keep naming all in camel style. productId, wantedAmount

Reply:
Done